frontend/src/components/RepositoryVCSProviderPanel.vue

```diff
@@ -79,6 +79,9 @@ export default {
     });
 
     const eventListener = (event: Event) => {
+      event.stopImmediatePropagation();
+      event.preventDefault();
```

Comment:
why do we need these 2 extra lines?

Reply:
If there are multiple event listeners, the first line will ensuer only one handler will be executed, the second will preevent  the default behavior (which happens to be none in this case).

Reply:
I thought there should be only one listener for a particular event? If there are multiple, it seems to be a problem with our code?